src/MEArec/generators/recordinggenerator.py

```diff
@@ -1410,9 +1410,9 @@ def generate_recordings(
 
             # compute pad samples as 3 times the low-cutoff period
             if cutoff.size == 1:
-                pad_samples_filt = 3 * int((1.0 / cutoff * fs).magnitude)
+                pad_samples_filt = 3 * int((1.0 / cutoff * fs).magnitude[0])
```

Comment:
Numpy won't cast an array to a scalar in future versions so the value must be pulled out of the array. Unless you know that quantities plans to update to change their behavior in which case this PR can be closed.

Reply:
I think it'd be safer to do it here :) thanks!

Reply:
same here! because cutoff is  already scalar
```suggestion
                pad_samples_filt = 3 * int((1.0 / cutoff * fs).magnitude)
```

Reply:
actually I think the problem is that cutoff might be an array because the testing warned that there was an array somewhere. I'm going to try to test one other thing really quick. But can cutoff itself ever be a scalar or will it always be an array?

Reply:
Let me check :)

Reply:
So the difference is that `test_cli` leads to an array warning which will become an error 
whereas `test_gen_recordings_filters` leads to 0-d array that creates a hard error currently. Maybe there needs to be one more elif in this control flow to account for the differences?

Reply:
Just so we have the `test_cli` warning in front of us:
```
src/MEArec/tests/test_generators.py::TestGenerators::test_cli
  /home/runner/work/MEArec/MEArec/src/MEArec/generators/recordinggenerator.py:1413: DeprecationWarning: Conversion of an array with ndim > 0 to a scalar is deprecated, and will error in future. Ensure you extract a single element from your array before performing this operation. (Deprecated NumPy 1.25.)
    pad_samples_filt = 3 * int((1.0 / cutoff * fs).magnitude)
```

Reply:
![image](https://github.com/SpikeInterface/MEArec/assets/17097257/5265856a-6d44-4f17-b86c-9851d3e5b563)

I think that in the python API you can have a scalar cutoff for high-pass. I'll check the CLI

Reply:
found the issue, can I push to your PR?

Reply:
Sure fix away!